### ISSUE TITLE
PHP7 testing issues

### ISSUE THREAD

**Issue by AlexMiroshnikov** [Mufuphlex/cplt#2]
``` php
Failed asserting that exception of type "TypeError" matches expected exception "\PHPUnit_Framework_Error". Message was: "Argument 1 passed to Mufuphlex\Cplt\ContainerItem::setAncestors() must be of the type array, string given, called in /home/travis/build/Mufuphlex/cplt/tests/ContainerItemTest.php on line 66"
```

**Comment by AlexMiroshnikov** [comment]
https://github.com/sebastianbergmann/phpunit/issues/1778

**Comment by AlexMiroshnikov** [comment]
Resolved in #3 .
